### PR TITLE
refactor: replace `execa` with `tinyexec`

### PR DESCRIPTION
**Pull request by beeequeue**
- Replaced `execa` with `tinyexec`
- Updated all `picocolors` imports to use the default export

### CODE REVIEW

**Comment by sxzz** [comment]
I'm considering replacing execa with [tinyexec](https://github.com/tinylibs/tinyexec).

**Comment by beeequeue** [comment]
I can try my hand at it 🙂

**Comment by beeequeue** [comment]
I assume I should revert the ESM change if possible?

File: src/lib/report-size.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as colors from 'picocolors'
+import colors from 'picocolors'
```
Comment (beeequeue) [inline src/lib/report-size.ts:1]: I was getting `colors.* is not a function` errors running in every environment, and while logging it showed that everything was inside the default export.

**Review by sxzz — APPROVED** [review]

**Review by sxzz — APPROVED** [review]

**Review by sxzz — APPROVED** [review]

**Pull request merged**